monitoring/rid_qualifier/flight_track_factory.py

```diff
@@ -0,0 +1,178 @@
+        # Now we have a point, we can buffer the point and create bounding boxes of the buffer to get the appropriate polygons, more than three boxes can be created, for the tests three will suffice.
+        box_diagonals = {1:{'length':150, 'name':'zoomed_in_detail'},2:{'length':380, 'name':"whole_flight_area"},3:{'length':3000, 'name':'too_large_query'}}
+
+        for box_id, box_diagonal in box_diagonals.items():
+            # Buffer the point with the appropriate length
+            buffer = pt.buffer(box_diagonal['length'])                   
+            buffer_bounds = buffer.bounds
+            buffer_bounds_polygon  =  shapely.geometry.box(buffer_bounds[0], buffer_bounds[1], buffer_bounds[2], buffer_bounds[3])
+            buffer_points =  zip(buffer_bounds_polygon.exterior.coords.xy[0], buffer_bounds_polygon.exterior.coords.xy[1])
+            proj_buffer_points = []
+            # reproject back to ESPG 4326
+            transformer2 = Transformer.from_crs("epsg:3857", "epsg:4326")
+            for point in buffer_points:
+                x = point[0]
+                y = point[1]
+                x, y = transformer2.transform(x, y)
+                proj_buffer_points.append((x, y))
+            
+            buffered_box = Polygon(proj_buffer_points)
+            # Get the bounds of the buffered box, this is the one that will will be fed to the remote ID display provider to query
+            self.query_bboxes.append({'name':box_diagonals[box_id]['name'], 'shape': buffered_box})
+            
+
+    def generate_flight_grid(self):
+        ''' Generate a series of boxes within the bounding box to have areas for different flights '''
+        # Compute the box where the flights will be created. For a the sample bounds given, over Bern, Switzerland, a division by 2 produces a cell_size of 0.0025212764739985793, a division of 3 is 0.0016808509826657196 and division by 4 0.0012606382369992897. As the cell size goes smaller more number of flights can be accomodated within the grid. For the study area bounds we build a 3x2 box for six flights by creating 3 column 2 row grid. 
+
+        cell_size_x = (self.maxx - self.minx)/(3) # create three columns
+        cell_size_y = (self.maxy - self.miny)/(2) # create two rows
+        grid_cells = [] 
+        for u0 in range(0, 3):  # 3 columns           
+            x0 =  self.minx + (u0 * cell_size_x)            
+            for v0 in range(0,2): # 2 rows
+                y0 = self.miny + (v0 *cell_size_y)
+                x1 = x0 + cell_size_x
+                y1 = y0 + cell_size_y
+                grid_cells.append(shapely.geometry.box(x0, y0, x1, y1))                
+        
+        self.flight_grid = grid_cells
+        
+
+    def generate_flight_paths_points(self):
+        ''' For each of the boxes allocated to the operator, get the centroid and buffer to get a flight path. A 75 m radius is provided to have flight paths within each of the boxes '''
```

Comment:
In this method, the documentation at the top says 75 m, the documentation further down says 100 m, and the code and accompanying comment has 70.  I recommend only mentioning the value once, in code, where it's defined.  So, e.g.:

```python
RADIUS = 70 # Radius of each flight's circular flight path (meters)
```

Reply:
The 70m radius is the correct one, I have checked and will make the modifications in the document.